Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -241,11 +241,18 @@ module Kitos.Utility.KendoGrid {
         Right
     }
 
+    export enum KentoToolbarImplementation {
+        Button,
```

Comment:
@JacobMalling Her blev der behov for at udvide toolbar typen og senere kommer også dropdown til. Måden vi så gør det på er at udvide typerne generisk og så lave optional params som input til implementation details